Basic-Captions/css/app.css

```diff
@@ -0,0 +1,44 @@
+body, html {
+    background-color: gray;
+    height: 100%;
+}
+#videos {
+    position: relative;
+    width: 100%;
+    height: 100%;
+    margin-left: auto;
+    margin-right: auto;
+}
+#subscriber {
+    position: absolute;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    z-index: 10;
+}
+#publisher {
+    position: absolute;
+    width: 360px;
+    height: 240px;
+    bottom: 10px;
+    left: 10px;
+    z-index: 100;
+    border: 3px solid white;
+    border-radius: 3px;
+}
+#buttonholder{
+    position: absolute;
+    width: 70%;
+    margin-left: auto;
+    margin-right: auto;
+    bottom: 2px;
+    left: 20px;
+    z-index: 200;
+}
+.caption-box {
```

Comment:
I think the caption looked a little small when we were troubleshooting, but I was only seeing it via screenshare so I will leave the judgement call up to you. It's obviously not important or a blocker at all.

Reply:
I moved the publisher to the top left instead of the bottom left so that the Publisher object does not block the captions as they come in, but I kept the font size the same; I feel like it's big enough

Reply:
Awesome, thanks man